components/select/tuya.rst

```diff
@@ -50,12 +50,15 @@ Configuration variables:
 ------------------------
 
 - **enum_datapoint** (**Required**, int): The enum datapoint id number for the select.
+  At least one of *enum_datapoint* or *int_datapoint* is required.
+- **int_datapoint** (**Required**, int): The int datapoint id number for the select.
+  At least one of *enum_datapoint* or *int_datapoint* is required.
 - **options** (**Required**, Map[int, str]): Provide a mapping from values (int) of
   this Select to options (str) of the *enum_datapoint* and vice versa. All options and
   all values have to be unique.
 - **optimistic** (*Optional*, boolean): Whether to operate in optimistic mode - when in this mode,
   any command sent to the Select will immediately update the reported state.
-
+- **data_type** (*Optional*, enum): Set the backing data type, one of: ``enum`` or ``int``. Defaults to ``enum``.
```

Comment:
Not required right?
```suggestion
```

Reply:
Correct, forgot to update documentation after PR refactor.